javascript/javascript1/week2/Student Manager/studentmanage.js

```diff
@@ -0,0 +1,29 @@
+const class07Students = [];
+function addStudentToClass(studentName) {
+  if (studentName === "Queen" || studentName === "queen") {
```

Comment:
Is there any other way of doing this comparison? Hint: string.toLowercase() :)

Reply:
Thanks, I have converted the code using toLowerCase() method.